src/components/books/list.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars */
+import Books from './books.js';
+import './list.css';
+
+function List() {
+  const bookList = [{
+    title: 'Republic',
+    author: 'Plato',
+    genre: 'Phylosophy',
+    id: 1,
+  },
+  {
+    title: 'The Prince',
+    author: 'Machiavel',
+    genre: 'Phylosophy',
+    id: 2,
+  }];
+
+  return (
+        <ul>
+            {bookList.map((book) => (
+               <Books name={book.title} author={book.author} genre={book.genre} key={book.id}/>
+            ))}
+        </ul>
+  );
```

Comment:
* Like the previous reviewer pointed out you should have a component that Renders a form with inputs for the title and the author of the book with a submit button but the button isn't expected to work for now just needs to be rendered
* Refer to project sneak peak 
![image](https://user-images.githubusercontent.com/52497246/130642369-5148d970-6ade-42fa-bec7-49c68f8482d7.png)